app/jobs/profile_proofing_components_backfill_job.rb

```diff
@@ -0,0 +1,33 @@
+class ProfileProofingComponentsBackfillJob < ApplicationJob
+  queue_as :long_running
+
+  def perform
+    Profile.find_in_batches do |batch|
```

Comment:
This wouldn't be an incredible amount of savings since it's unindexed, but would save some Ruby time if we filter out the rows that don't need to be updated at the beginning with something like:

```sql
WHERE proofing_components->'id' IS NULL
```

Reply:
sure, gave it a shot in 12a3a6a66a451aa41ad18c3d8588324df9eba3fb